ci-operator/config/redhat-appstudio/rhtap-cli/redhat-appstudio-rhtap-cli-main.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  rhtap-installer:
+    name: redhat-appstudio
+    namespace: ci
+    tag: rhtap-installer
+build_root:
+  project_image:
+    dockerfile_path: ci/Dockerfile
+images:
+- dockerfile_path: ci/Dockerfile
+  from: src
+  to: rhtap-cli
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: rhtap-cli-install
```

Comment:
could you use name `rhtap-cli-install-nightly`?  So that we can identify is as a periodic job

Reply:
Yes I can change this though this job will change, right now I'm just trying to get the install and e2e tests to work.

Then I'll be using the rhtap-cli workflow in the gitops and rhdh pre-release jobs.

I can also create a nightly that just runs rhtap-cli with rhtap e2e tests but I don't think we need that as we are getting that coverage in Konflux now right?

Reply:
Konflux doesn't support cron job :( We still need to run these cron jobs in Openshift CI

Reply:
Ah ok so we can still run the rhtap-cli on a cron even though the same job will also be running on PRs to the repo in Konflux.